easybuild/easyconfigs/r/REViewer/reviewer.CMakeLists.txt.patch

```diff
@@ -0,0 +1,11 @@
+--- CMakeLists.txt.orig	2025-03-10 14:47:29.661273709 +0100
++++ CMakeLists.txt	2025-03-10 14:47:39.773327546 +0100
```

Comment:
```suggestion
Recent versions of Boost requires C++17 so we enforce it for the build

--- CMakeLists.txt.orig	2025-03-10 14:47:29.661273709 +0100
+++ CMakeLists.txt	2025-03-10 14:47:39.773327546 +0100
```

Reply:
The new comments will require updating the checksums

Reply:
Thanks for the corrections. I'll try to fix the checksums locally and push.